### DISCUSSION TITLE
Cant import CSV's

### DISCUSSION THREAD

**Discussion by ghost** [netbox-community/netbox#10323] [Q&A]
Guys, I'm at a loss here.

https://github.com/netbox-community/netbox/issues/10321

Read this issue. Please help me understand how this is NOT a bug?
You go to export a CSV, then select the file CSV import and you get an error.

**Comment by jeremystretch** [comment]
Your questions have been answered in the issue. Please don't open duplicate discussions because you disagree with the response.
